doc/source/user/quickstart.rst

```diff
@@ -1054,7 +1056,7 @@ element is assumed to be the same along that dimension for the
 "broadcast" array.
 
 After application of the broadcasting rules, the sizes of all arrays
-must match. More details can be found in :doc:`basics.broadcasting`.
+must match. More details can be found in :ref:`basics.broadcasting`.
```

Comment:
This link wasn't broken either - the `:doc:` role is preferred for linking to documents I think.

Reply:
Please see my comment below: https://github.com/numpy/numpy/pull/16472#issuecomment-639450633